runtime/devnet/src/config/assets.rs

```diff
@@ -30,6 +30,8 @@ parameter_types! {
 parameter_types! {
 	pub NftsPalletFeatures: PalletFeatures = PalletFeatures::all_enabled();
 	pub const NftsCollectionDeposit: Balance = 10 * UNIT;
+	// Key = 116 bytes (4+16+32+16+32+16), Value = 17 bytes (1+8+8)
+	pub const NftsCollectionApprovalDeposit: Balance = deposit(1, 133);
```

Comment:
Not essential, but a test using `KeyLenOf` and `(Option<BlockNumberFor<T>>, DepositBalanceOf<T, I>)::max_encoded_len()` could be used to check this value is sane. The former may require a storage alias defined in the test to realise.

Not a blocker.

Reply:
There is a test `ensure_collection_approval_bytes` in `tests.rs` that validate the bytes. Not sure if this is what you mean.